File: San353/app/src/main/java/ca/six/san3/MainActivity.java
package ca.six.san3;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import ca.six.san3.core.BaseActivity;
import ca.six.san3.module.game.GameActivity;
import ca.six.san3.utils.JumpUtils;

public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btnMainStart).setOnClickListener(v -> {
            JumpUtils.go(this, GameActivity.class);
        });
    }
}

File: San353/app/src/main/java/ca/six/san353/MainActivity.kt
package ca.six.san353

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import ca.six.san353.biz.game.GameActivity
import ca.six.san353.utils.jump
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnMainStart.setOnClickListener { view -> jump(GameActivity::class.java) }

    }
}
